readthedocsext/theme/templates/allauth/layouts/base.html

```diff
@@ -2,6 +2,13 @@
 
 {% block content-wrapper %}
   <div class="ui very padded container">
+
+    {# This is overriding the normal base.html content-wrapper, #}
+    {# so we need to include notifications here so they're displayed. #}
+    {% block notifications %}
+      {% include "includes/utils/messages.html" %}
```

Comment:
This at least avoids duplicating logic

```suggestion
      {{ block.super }}
```